cfme/containers/provider/openshift.py

```diff
@@ -145,15 +145,15 @@ def view_value_mapping(self):
                 'image_tag': self.image_tag,
                 'cve_loc': self.cve_loc
             }
+        elif self.appliance.version >= '5.10':
```

Comment:
**REQUIRED:** Forgive me, but this won't fire will it? If the appliance is '5.10' then it will hit the first conditional, as it's also over '5.9' and won't actually do what you want. Also is it the case that if it's over '5.10' you don't want any of the other fields populated? So if it worked as I believe you intended, you woulnd't ever see ```mapping['alerts_type']``` being set.